lib/rewards/server/subscribers/object-subscriber.js

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * Reldens - Rewards Object Subscriber
+ *
+ */
+
+
+class ObjectSubscriber
```

Comment:
as general rule I was thinking on rename these because are not actual subscribers: that would be the case if we pass the event handler and inside of these classes we include the `event.on('some-event', handler.doThis());`

in this case these are more like "handlers", a more appropiated name would be ObjectEnrichRewardsHanlder, which would contain the "execute()" method, that will enrich the objectInstance with the rewards.

for now don't change it, but keep this notes in mind because I may refactor that part with all the other subscribers that are not actual subscribers at once

Reply:
ok!